services/common/rs_server_common/utils/utils.py

```diff
@@ -49,33 +49,47 @@ def is_valid_date_format(date: str) -> bool:
     False
     """
     try:
-        datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ")
+        datetime.strptime(date, "%Y-%m-%dT%H:%M:%SZ")
         return True
     except ValueError:
         return False
 
 
-def validate_inputs_format(start_date, stop_date):
+def validate_inputs_format(interval: str) -> Tuple[datetime | None, datetime | None, int | None, str | None]:
     """
-    Validate the date format of start_date and stop_date.
+    Validate the format of the input time interval.
 
-    Parameters:
-    - start_date (str): The start date to be validated.
-    - stop_date (str): The stop date to be validated.
+    This function checks whether the input interval has a valid format (start_date/stop_date) and
+    whether the start and stop dates are in a valid ISO 8601 format.
+
+    Attributes:
+    - interval (str): The time interval to be validated, with the following format:
+      "2024-01-01T00:00:00Z/2024-01-02T23:59:59Z"
 
     Returns:
-    tuple[bool, JSONResponse]: A tuple containing a boolean indicating the validity of the date format
-    and a JSONResponse instance. If the date format is invalid, the boolean is False, and the JSONResponse
-    contains a 400 Bad Request response with an appropriate error message. If the date format is valid,
-    the boolean is True, and the JSONResponse is None.
+        Tuple[datetime | None, datetime | None, int | None, str | None]:
+            A tuple containing:
+            - start_date (datetime): The start date of the interval.
+            - stop_date (datetime): The stop date of the interval.
+            - err_code (Optional[int]): An HTTP status code indicating an error (or None if no error).
+            - err_text (Optional[str]): An error message (or None if no error).
+
+    Note:
+        - The input interval should be in the format "start_date/stop_date"
+        (e.g., "2022-01-01T00:00:00Z/2022-01-02T00:00:00Z").
+        - This function checks for missing start/stop and validates the ISO 8601 format of start and stop dates.
+        - If there is an error, err_code and err_text provide information about the issue.
     """
+
+    try:
+        start_date, stop_date = interval.split("/")
+    except ValueError:
+        return None, None, status.HTTP_422_UNPROCESSABLE_ENTITY, "Missing start/stop"
```

Comment:
It would be easier here to just:
```python
from fastapi import HTTPException
raise HTTPException (status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail="Missing start/stop")
```
instead of returning err_code, err_text and return a JSONResponse from the caller.